VirtualTables.sql

```diff
@@ -0,0 +1,161 @@
+CREATE VIEW Tms_TempFrame AS
+(
+SELECT firstTemp.val  AS TempSensorOne,
+       secondTemp.val AS TempSensorTwo,
+       thirdTemp.val  AS TempSensorThree,
+       fourthTemp.val AS TempSensorFour,
+       firstTemp.contextId,
+       firstTemp.receiveTime
+FROM TmsSensorTemp AS firstTemp
+         FULL JOIN (SELECT val, receiveTime, contextId FROM TmsSensorTemp WHERE sensorId = 2) AS secondTemp
+                   ON firstTemp.receivetime = secondTemp.receivetime AND firstTemp.contextid = secondTemp.contextId
+         FULL JOIN (SELECT val, receiveTime, contextId FROM TmsSensorTemp WHERE sensorId = 3) AS thirdTemp
+                   ON firstTemp.receivetime = thirdTemp.receivetime AND firstTemp.contextid = thirdTemp.contextId
+         FULL JOIN (SELECT val, receiveTime, contextId FROM TmsSensorTemp WHERE sensorId = 4) AS fourthTemp
+                   ON firstTemp.receivetime = fourthTemp.receivetime AND firstTemp.contextid = fourthTemp.contextId
+WHERE sensorId = 1
+    );
+
+CREATE VIEW Tms_Fan_Frame AS
+(
+SELECT firstFan.val AS fanSensorOne, secondFan.val AS fanSensorTwo, firstFan.contextId, firstFan.receiveTime
+FROM TmsFanSpeed AS firstFan
+         FULL JOIN (SELECT val, receiveTime, contextId FROM TmsFanSpeed WHERE fanid = 2) AS secondFan
+                   ON firstFan.receivetime = secondFan.receivetime AND firstFan.contextid = secondFan.contextId
+WHERE fanId = 1
+    );
+
+CREATE VIEW Imu_Accel_Frame AS
```

Comment:
Can we add a comment before each view briefly summarizing it?

Reply:
Also, what do you think about just making one giant IMU table with all the IMU data?

Reply:
Will add the comments. I think if we were to make a big table it would have to be built on the views I have created unless there are rules against that